doc/api/crypto.md

```diff
@@ -1,5 +1,7 @@
 # Crypto
 
+<!--introduced_in=v0.10.0-->
```

Comment:
I don't believe this is correct. (https://nodejs.org/docs/v0.3.6/api/crypto.html) `crypto` has been around for a while. https://nodejs.org/en/download/releases/ is a useful page to check :]